VogtFactory/RentalCar.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtFactory
+{
+    class RentalCar
+    {
+        string name;
+        int numberOfPeopleThatCanFit;
+
+        RentalCarFactory rcf = new RentalCarFactory();
+
+        public RentalCar orderCar(string type)
```

Comment:
Functions are always PascalCase. This should be `OrderCar(string type)`

Reply:
This `OrderCar` function shouldn't be a part of the base Rental car.

Get rid of this function, and I'll go post my next comment in your `Run()` function to show you how to do this thereee